demos/flowchart.html

```diff
@@ -1540,6 +1540,30 @@ <h3>flowchart</h3>
     </pre>
     <hr />
 
+    <pre class="mermaid">
+      ---
+      title: Subgraph nodeSpacing and rankSpacing example
+      ---
+      %%{init:{'flowchart':{'nodeSpacing': 1, 'rankSpacing':'1'}}}%%
```

Comment:
Avoid directives configuration, which is obsolete, use frontmatter
```suggestion
      ---
      title: Subgraph nodeSpacing and rankSpacing example
      config:
        flowchart:
          nodeSpacing: 1
          rankSpacing: 1
      ---
```